utils/http_test.go

```diff
@@ -66,3 +66,39 @@ func TestUnitGetPractitionerIDFromVars(t *testing.T) {
 		So(PractitionerID, ShouldBeEmpty)
 	})
 }
+
+func TestUnitGetAttachmentIDFromVars(t *testing.T) {
+	Convey("Get Attachment ID", t, func() {
+		vars := map[string]string{
+			"attachment_id": "67890",
+		}
+		PractitionerID := GetAttachmentIDFromVars(vars)
```

Comment:
Should this be `attachmentID` instead?

```suggestion
                attachmentID := GetAttachmentIDFromVars(vars)
		PractitionerID := GetAttachmentIDFromVars(vars)
```